sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert (year / week).evalf(15) == Float(52.17745571428571)
```

Comment:
It indeed does not add noticeable time, but (a) it's a precedent for the bad practice of routinely adding test cases that do not really test anything, (b) the Float constructor is given a Python float so the argument that comparing with a Float is precise falls down (and may indeed fail on architectures with different IEEE architecture such as ARM, or faulty/configuration-dependent IEEE like i386), (c) if the test fails, and it's not something in the definition of ```week``` (as is most likely), the test failure is going to be a red herring for the maintainer.

It's of minimal value against regressions I think; any regression should be caught by a unit test that was written when the potentially recurring bug was fixed.

This does indeed teach the technicalities of adding a unit test, and as such, it does server its purpose.
It fails at teaching how to write *useful* unit tests though. And it's introducing problems in the test codebase where there was none before (the unexplained float literal, and the architecture-dependent float->Float conversion).